### PR TITLE
TSE Examples added (Allan Alvarado)

### PR DESCRIPTION
**Pull request by Cygnum16**
These are examples created by different TSEs from NI.

### CODE REVIEW

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
+"""
+Python DAQmx API program:
+	ContGen_IntClk
+
+Author: Andres Quesada
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:5]: Everything in this repo should inherit the [LICENSE](https://github.com/ni/nidaqmx-python/blob/master/LICENSE). Let's remove these author/copyright blocks.

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+  try:
+    sys.exit(0)
+  except SystemExit:
+    os._exit(0)
+
+def print_error_info(error):
+  print("DAQmx error", error.error_code, "occurred")
+  print("Cause: ", error.error_type)
+
+def ConfigureSampleClkTiming(desiredFreq, sampsPerbuff, cyclesPerBuff, DAQmxTask,sampsPerCycle):
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:96]: this function name isn't pythonic. can you make it `cfg_samp_clk_timing`?

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+  Outputs:
+    desiredSampClkRate: Is the calculated sample rate to be used at the DAQmx timing configuration.
+    resultingFrequency: I DON'T KNOW WHAT THIS WAS USED FOR 
+  """
+  sampsPerCycle = sampsPerbuff / cyclesPerBuff
+  desiredSampClkRate = desiredFreq * sampsPerbuff / cyclesPerBuff
+  resultingFrequency = desiredSampClkRate / sampsPerCycle
+
+  try:
+    DAQmxTask.timing.cfg_samp_clk_timing(desiredSampClkRate, sample_mode = AcquisitionType.CONTINUOUS)
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:117]: `DAQmxTask` is a confusing name. I thought this was calling a static function on a class. Can you rename the variable `task` to match the rest of the program?

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py (hunk 2 above)
Comment (zhindes) [reply nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:96]: Same is true for all of these functions in all of these examples.

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+    task.start()
+  except DAQmxError.DaqError as err:
+    print_error_info(err)
+    exit_application()
+
+  #Generation loop
+  try:
+    try:
+      while(not task.is_task_done()):
+        print("Generating Data")
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:250]: you should tell them how to stop. it looks like ctrl+c?

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+
+Example Category:
+    Analog Generation
+
+ Description:
+    This example demonstrates how to output a continuous number of
+    current samples to an Analog Output Channel using an internal
+    sample clock.
+
+ Instructions for Running:
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:16]: These instructions sound like there is an interactive UI or something. Can you change the language to make it clear that they need to modify the code?

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+  Cycles per buffer and desired Waveform Frequency. It also sets the Acquisition Mode (Finite / Continuous)
+  and the samples per channel.
+
+  Inputs:
+    desiredFreq : Is the desired Frequency for the selected Waveform [Hz]
+    sampsPerBuff: Number of samples to write in the output buffer
+    cyclesPerBuff: Number of cycles within a buffer write operation
+    sampsPerCycle: Number of samples to write on a single cycle
+
+  Outputs:
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:108]: there are no outputs here...

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+
+      if percentPeriod <= 0.25:
+        data.append(offset + dat)
+      elif percentPeriod <= 0.75:
+        data.append(offset + (2.0 * amplitude - dat))
+
+      else:
+        data.append(offset + (dat - 4.0 * amplitude))
+
+  phase = math.fmod(phase + frequency * 360 * numElements, 360)
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:183]: nothing uses phase after this... what is this doing? was it meant to be in the loop?

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+
+
+def GenerateWaveform(WaveformType,sampsPerCycle):
+  
+  if WaveformType =='Triangle':
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:188]: Python has enums, you should use that

File: nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+      time.sleep(0.01)
+   
+  except KeyboardInterrupt:
+    #-------------------------------------
+    # DAQmx End Code
+    #-------------------------------------
+    task.stop()
+    exit_application()
+
+
```
Comment (zhindes) [inline nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py:263]: lots of empty lines

**Review by zhindes — CHANGES_REQUESTED** [review]
I only reviewed one example. I assume most of my comments apply to all of them.

A few other things:
* remove the placeholder.txt files from your commit
* if you're going to add a directory hierarchy, please move the existing examples
* match the naming of the existing examples, e.g. `snake_case.py`

**Comment by Cygnum16** [comment]
Thank you for your input, Zach! I will check again all the examples with your feedback provided and once they are correct I will create the pull request again. 

Thank you again!

**Comment by zhindes** [comment]
Closing due to inactivity

**Pull request closed without merging**